src/codewhisperer/explorer/codewhispererNode.ts

```diff
@@ -77,6 +78,8 @@ export class CodeWhispererNode implements RootNode {
             return AuthUtil.instance.isEnterpriseSsoInUse()
                 ? 'IAM Identity Center Connected'
                 : 'AWS Builder ID Connected'
+        } else if (AuthUtil.instance.isConnectionExpired()) {
+            return 'Expired Connection'
```

Comment:
'Expired Connection' vs 'Connection Expired' ?

Reply:
according to the figma it's 'Expired Connection'

Reply:
https://www.figma.com/file/csYNy1ncFzR7FZFJBak5UK/CodeWhisperer?node-id=5262%3A295516&t=I7US3sGcevHng6a7-0